server/src/main/java/org/apache/iotdb/db/writelog/manager/MultiFileLogNodeManager.java

```diff
@@ -46,6 +46,12 @@
 public class MultiFileLogNodeManager implements WriteLogNodeManager, IService {
 
   private static final Logger logger = LoggerFactory.getLogger(MultiFileLogNodeManager.class);
+  // if OOM occurs when registering bytebuffer, getNode method will sleep awhile and then try again
+  private static final int REGISTER_BUFFER_SLEEP_INTERVAL = 20;
+  // if total sleep time exceeds this, getNode method will reject this write and throw original
+  // exception.
+  private static final int REGISTER_BUFFER_REJECT_THRESHOLD = 1_000;
```

Comment:
Make these two threshold configurable, and indicate theirs unit in the comment, like `ms` or `s`.

Reply:
I add ms unit for them. But, I think it's bad to make these threshold params configurable, because they may make users confused.